Python/GD_Content_Downloader/Readme.md

```diff
@@ -0,0 +1,39 @@
+# Google Drive Content Downloader
+
+The Script download the Folder or file with the link(has view permission) from GoogleDrive
+
+## Requirement
+ You need A credentials.json file with access to the google drive API
```

Comment:
Add backticks around `credentials.json` and add proper full-stop. Remove unnecessary capitalization as well.